### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by Pen-y-Fan** [tailwindtoolbox/Admin-Template#4]
Many thanks for sharing this template, it has given me a lot of ideas for my project. 

Before I start my project I needed to improve the accessibility, I aim for [WCAG AA](https://www.w3.org/TR/WCAG21/) standard. Lighthouse and Axe dev tools have been used to get 100% and 0 issues. There are still possible improvements to be made, but this is a step in the right direction.

Technically the aside and footer should be two separate elements, even though they are the same content, as the tab order will be incorrect for keyboard users.  This would be easier if extracted to a component, so I've left it for this HTML template.

I'll submit the PR shortly.

I hope this will help you and others.

**Comment by tailwindtoolbox** [comment]
Merged! Thanks again!
